client/entrypoints/payment-request-settings/__tests__/payment-request-button-preview.test.js

```diff
@@ -25,6 +25,7 @@ jest.mock( '@stripe/react-stripe-js', () => ( {
 } ) );
 
 jest.mock( 'wcstripe/data', () => ( {
+	usePaymentRequestEnabledSettings: jest.fn(),
```

Comment:
Does this make a difference in the JS tests execution?

Reply:
Good catch, `usePaymentRequestEnabledSettings ` is not used in those tests.